angular/src/app/shared/data-table/data-table.component.html

```diff
@@ -1,4 +1,4 @@
-<ag-grid-angular class="ag-theme-balham"
+<ag-grid-angular [ngClass]="['ag-theme-balham', hideBorder ? 'hide-border' : '']"
```

Comment:
More refined way how to achieve this would be using:
```
                 [class.hide-border]="hideBorder"
                 class="ag-theme-balham"
```

This will do the same, but is easier to read.